### PR TITLE
README and misc update

### PR DESCRIPTION
**Pull request by nicolasburtey**

### CODE REVIEW

File: src/servers/graphql-server.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const isAuthenticated = rule({ cache: "contextual" })((parent, args, ctx)
 
 export const isApiKeyAuthenticated = rule({ cache: "contextual" })(
   (_parent, _args, ctx) => {
-    return ctx.account !== null ? true : "NOT_AUTHENTICATED"
+    return ctx.account !== null ? true : "NOT_AUTHENTICATED_APIKEY"
```
Comment (krtk6160) [inline src/servers/graphql-server.ts:40]: What's the context around this change?

**Review by krtk6160 — COMMENTED** [review]
One question otherwise LGTM

File: src/servers/graphql-server.ts (hunk 1 above)
Comment (nicolasburtey) [reply src/servers/graphql-server.ts:40]: there are issues related to nginx in prod. and the string is `NOT_AUTHENTICATED`. so making this string unique with the previous condition will help to understand where the issue is

**Review by krtk6160 — CHANGES_REQUESTED** [review]
Line 153 in `graphql-server.ts` needs to be changed also I think. The isShieldError function.

**Review by bodymindarts — DISMISSED** [review]

**Review by dolcalmi — DISMISSED** [review]

**Review by krtk6160 — APPROVED** [review]

**Review by bodymindarts — APPROVED** [review]

**Pull request merged**